core/class.go

```diff
@@ -79,6 +100,16 @@ func (c *Cairo1Class) Hash() *felt.Felt {
 	)
 }
 
+func (c *CompiledClass) CompiledClassHash() *felt.Felt {
```

Comment:
We can rename this to `Hash` to remove the `CompiledClass.CompiledClassHash()` stutter I think.

Reply:
Totally agree.